packages/components/src/focal-point-picker/index.tsx

```diff
@@ -108,17 +176,22 @@ export default function FocalPointPicker( {
 		return getFinalValue( { x: nextX, y: nextY } );
 	};
 
-	const getFinalValue = ( value ) => {
+	const getFinalValue = ( value: FocalPointType ): FocalPointType => {
 		const resolvedValue = resolvePoint?.( value ) ?? value;
 		resolvedValue.x = Math.max( 0, Math.min( resolvedValue.x, 1 ) );
 		resolvedValue.y = Math.max( 0, Math.min( resolvedValue.y, 1 ) );
+		const roundToTwoDecimalPlaces = ( n: number ) =>
+			Math.round( n * 1e2 ) / 1e2;
```

Comment:
Actually @stokesman, the unit tests are expecting string values to be returned, I'm wondering if there are good reasons to keep it that way, or if it's ok to return number from now on?

Reply:
Seems like it should be okay as a number. #18765 was where the use of `toFixed` was introduced and indeed seems only to have been for the sake of rounding.

Reply:
Done in 08a71e98a50af41cab272ec8a86440d0a1b476d5

I did some manual testing and added unit tests to confirm that:

- It still works when given a string value initially (because Gutenberg blocks will already have string values saved)
- When a string value is given initially, it will not fire an `onChange` until the value is actually changed by the user (this is to ensure that the post content will not be marked as dirty just by loading existing instances of the Cover block)

> In one of the unit tests failures, the returned value is `0` instead of a string floating number — so we may need to also double check the new rounding logic

I confirmed that this is the case on trunk as well (first call to `onChange` receives 0). I tweaked the assertions so it only checks the last call, as that may be a bit confusing when the test fails.